app/src/main/java/com/project/lab1/MainActivity.kt

```diff
@@ -1,62 +1,49 @@
 package com.project.lab1
 
-import android.app.AlertDialog
-import android.content.DialogInterface
 import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.util.Log
-import android.widget.*
-import androidx.appcompat.app.AppCompatActivity
+import android.widget.TextView
+import com.project.lab1.feed.FeedActivity
+import com.project.lab1.feed.models.FeedItem
+import kotlinx.android.synthetic.main.activity_main.*
+import java.io.Serializable
 
-
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity(){
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
```

Comment:
Change init of activity view to android view binding https://developer.android.com/topic/libraries/view-binding